test/test_torch.py

```diff
@@ -15458,6 +15458,27 @@ def test_abs_signed_zero(self, device, dtype):
         for v in inp:
             self.assertGreater(math.copysign(1.0, v), 0.0)
 
+    @dtypes(torch.float)
+    def test_absolute(self, device, dtype):
```

Comment:
Did you lose the torch.absolute(t) (with no out kwarg) variant?

Reply:
It was never there, I can add it in.

Reply:
My mistake. Yep, let's add that just to be super thorough.